confgenerator/files.go

```diff
@@ -52,6 +53,10 @@ func (uc *UnifiedConfig) GenerateFilesFromConfig(ctx context.Context, service, l
 	case "": // Validate-only.
 		return nil
 	case "fluentbit":
+		// TODO(alex-basinov) add support for logging on BMS
+		if platform.FromContext(ctx).SkipLogging != true {
```

Comment:
I think this condition is backwards. (You're skipping when `SkipLogging` is false?)

Also, this is just going to skip config generation. Won't fluent-bit then fail to start because its config is missing?

Reply:
it's a leftover from testing which I forgot to update. 
Yes, it would fail to start. I haven't thought about it.

Ok, I removed the SkipLogging field and updated GenerateFluentBitConfigs() to generate a BMS-specific config if ResourceOverride is set to the BMSResource. 
PTAL ?